docs/shared_bindings_matrix.py

```diff
@@ -280,7 +251,7 @@ def frozen_modules_from_dirs(frozen_mpy_dirs, withurl):
     """
     frozen_modules = []
     for frozen_path in filter(lambda x: x, frozen_mpy_dirs.split(" ")):
-        source_dir = get_circuitpython_root_dir() / frozen_path[7:]
+        source_dir = get_circuitpython_root_dir() / frozen_path[6:]
```

Comment:
6 is the length of "../../". the old 7 was the length of "$(TOP)/". The difference is because in the new way of doing things, the value has undergone make variable-expansion.

Reply:
I think I would feel more comfortable with a `.remove_prefix()` (https://docs.python.org/3.9/library/stdtypes.html#str.removeprefix, Python 3.9 and later) or an `re` replace, since then what's going is explicit.